package.json

```diff
@@ -42,5 +42,6 @@
       "@tuyau/utils": "workspace:*"
     }
   },
-  "prettier": "@julr/tooling-configs/prettier"
+  "prettier": "@julr/tooling-configs/prettier",
+  "packageManager": "pnpm@9.12.2+sha512.22721b3a11f81661ae1ec68ce1a7b879425a1ca5b991c975b074ac220b187ce56c708fe5db69f4c962c989452eee76c82877f4ee80f474cebd61ee13461b6228"
```

Comment:
Can you just remove this field? I know this is a good practice, but I personally find it very annoying ahah

Reply:
No problem :)